go.mod

```diff
@@ -18,6 +18,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
```

Comment:
not sure what happened here, we don't really need to change go.mod at all for this change so you can just remove this file from the PR for now